tests/test_models.py

```diff
@@ -269,6 +269,7 @@ def test_zoom_for_res():
         extent, crs, identifier="MyCustomTmsEPSG3857", minzoom=6
     )
     assert tms.zoom_for_res(10) == 14
+    assert tms.zoom_for_res(5_000) == 6
```

Comment:
I'm unsure about this test, `5_000` is not a valid number so I would expect `zoom_for_res` to fail anyway 🤷 

can we add a proper test with an actual number?

Reply:
Hmm my understanding was that 5k m/pixel was the approximate resolution of zoom level 5 for EPSG:3857 (per [this chart](https://wiki.openstreetmap.org/wiki/Zoom_levels)), is that not right? 

`_` is a visual separator in number literals so `5_000` is equivalent to `5000` but I can update it if that's clearer

Reply:
> _ is a visual separator in number literals so 5_000 

oh 💩 I didn't know that 😂 

I think that's fine then!

Reply:
Ahh cool, I like to use it to make large numbers more readable